modules/microshift-accessing-cluster-locally.adoc

```diff
@@ -28,6 +28,13 @@ $ mkdir -p ~/.kube/
 $ sudo cat /var/lib/microshift/resources/kubeadmin/kubeconfig > ~/.kube/config
 ----
 
+. Update the permissions on your `~/.kube/config` file by running the following command: 
++
+[source,terminal]
+----
+$ chmod go-r ~/.kube/config
+----
```

Comment:
This same step is also needed in the section titled "Accessing the MicroShift cluster remotely from a workstation" after the step where the kubeconfig is copied to the local workstation.

Reply:
@dhellmann Hiya Doug! Thank you! I just updated that section as well